JSDemos/Demos/Common/EditorAppearanceVariants/React/App.js

```diff
@@ -1,229 +1,176 @@
-                width="100%"
-                placeholder="Select..."
-                label="State"
-                labelMode={this.state.labelMode}
-              >
-                <Validator>
-                  <RequiredRule />
-                </Validator>
-              </SelectBox>
-            </div>
-          </div>
-          <div className="center">
-            <TagBox
-              items={this.positions}
-              defaultValue={this.defaultPosition}
-              multiline={false}
-              stylingMode={this.state.stylingMode}
-              width="100%"
-              inputAttr={nameLabel}
-              placeholder="Select..."
-              label="Position"
-              labelMode={this.state.labelMode}
-            >
-              <Validator>
-                <RequiredRule />
-              </Validator>
-            </TagBox>
-          </div>
-          <div className="editors-container">
-            <div className="left">
-              <TextBox
-                stylingMode={this.state.stylingMode}
-                width="100%"
-                mask="+1 (000) 000-0000"
-                inputAttr={phoneLabel}
-                maskRules={this.phoneRules}
-                label="Phone"
-                labelMode={this.state.labelMode}
-              >
-                <Validator>
-                  <RequiredRule />
-                </Validator>
-              </TextBox>
-            </div>
-            <div className="right">
-              <DateBox
-                stylingMode={this.state.stylingMode}
-                width="100%"
-                placeholder="Select..."
-                label="Hire Date"
-                inputAttr={hireDateLabel}
-                labelMode={this.state.labelMode}
-              >
-                <Validator>
-                  <RequiredRule />
-                </Validator>
-              </DateBox>
-            </div>
-          </div>
-          <div className="center">
-            <TextArea
-              stylingMode={this.state.stylingMode}
-              defaultValue='Olivia loves to sell. She has been selling DevAV products since 2012.'
-              width="100%"
-              placeholder="Type..."
-              label="Notes"
-              labelMode={this.state.labelMode}
-              inputAttr={notesLabel}
-            />
-          </div>
-          <div className="center">
-            <Button
-              onClick={this.validateClick}
-              text="Save"
-              type="default"
-              className="validate"
-            />
-          </div>
-        </div>
-        <div className="options">
-          <div className="caption">Options</div>
+export default function App() {
+  const defaultStylingMode = 'outlined';
+  const [stylingMode, setStylingMode] = React.useState(defaultStylingMode);
+  const [labelMode, setLabelMode] = React.useState('static');
+  const changeStylingMode = React.useCallback(({ value }) => {
+    setStylingMode(value);
```

Comment:
We have an agreement to add a setter as a dependency. Although it's meaningless from technical point, it's can help some users avoid similar errors
```suggestion
    setStylingMode(value);
```

Reply:
fixed